ui-tests/cypress/support/pageObjects/content/management/AddPage.js

```diff
@@ -0,0 +1,82 @@
+import {htmlElements} from "../../WebElement";
+import Content from "../../app/Content.js";
+import AppPage from "../../app/AppPage.js";
+import ManagementPage from "./ManagementPage";
+
+export default class AddPage extends Content {
+
+  alertMessageDiv = `.alert`;
+  contentDescriptionInput = `${htmlElements.input}#description`;
+  contentTitleAttrInput = `.RenderTextInput`
+  contentAttrsItTab = `${htmlElements.a}#content-attributes-tabs-tab-it`;
+  contentTitleAttrInputIt = `attributes[0].values.it`
+  contentSaveButton = `${htmlElements.button}#saveopts`;
+  contentSaveButtonUl = `${htmlElements.ul}`;
+  contentSaveButtonWrapper = `${htmlElements.div}.StickySave__row--top`;
+  contentSaveButtonSaveAction = `${htmlElements.li}`;
+  contentAttrEnPane = `${htmlElements.div}#content-attributes-tabs-pane-en`;
+  contentAttrItPane = `${htmlElements.div}#content-attributes-tabs-pane-it`;
+
+  getAlertMessage() {
+    return this.getRoot().find(this.alertMessageDiv);
+  }
+
+  getTitleAttrItInput() {
+    return this.getContents().get(this.contentAttrItPane)
+               .find(this.contentTitleAttrInput).eq(0);
+  }
+
+  getTitleAttrEnInput() {
+    return this.getContents().get(this.contentAttrEnPane)
+               .find(this.contentTitleAttrInput).eq(0);
+  }
+
+  getDescriptionInput() {
+    return this.getContents()
+               .find(this.contentDescriptionInput);
+  }
+
+  getItLanguageTab() {
+    return this.getContents().find(this.contentAttrsItTab);
+  }
+
+  getSaveButton() {
```

Comment:
is this a get or an action?

Reply:
actually its get, although you see the click action inside it, it only clicks on Kebab menu so the list of actions becomes visible. Do you still think I should change something on this?

Reply:
yes, I think it would be better to get the kebab menu and then from there the menu elements

Reply:
refactored to DropDown class